desc/optimize/_constraint_wrappers.py

```diff
@@ -1018,7 +1027,7 @@ def jvp_unscaled(self, v, x, constants=None):
     @functools.partial(jit, static_argnames=("self", "op"))
     def _jvp_f(self, xf, dc, constants, op):
         Fx = getattr(self._constraint, "jac_" + op)(xf, constants)
-        Fx_reduced = Fx[:, self._unfixed_idx] @ self._Z
+        Fx_reduced = Fx @ self._D[:, self._unfixed_idx] @ self._Z
```

Comment:
I think this can be replaced with `self._unfixed_idx_mat` but I'm not certain and want another opinion